demo/edge-attachmediastream.js

```diff
@@ -3,7 +3,7 @@
  * And Edge current does not support re-negotiation for now in the SDK.
  */
 (function () {
-  if (window.webrtcDetectedBrowser === 'edge') {
```

Comment:
This was supposed to patch and prevent issues with Edge `15.15019` and above which doesn't seem to need this workaround for rendering audio and video separately. In the later versions of Edge, it appears the video to be black when audio+video is rendered together in the `<video>` element, but `<audio>` and `<video>` elements renders them properly.

After webrtc/adapter resolves the Edge patches for the latest releases, the users can drop-in the new AdapterJS version and use this SDK version to connect with Edge. But if we need to use legacy API for any reasons, we wait will for the proper AdapterJS update and update the SDK again.